wapitiCore/attack/mod_csp.py

```diff
@@ -0,0 +1,97 @@
+import requests, re
```

Comment:
You should respect import order convention https://stackoverflow.com/questions/22722976/import-order-coding-standard/22789161#22789161